openedx/features/course_experience/templates/course_experience/course-outline-fragment.html

```diff
@@ -91,11 +118,12 @@ <h3 class="section-title">${ section['display_name'] }</h3>
                                                             class="menu-icon icon fa fa-pencil-square-o"
                                                             aria-hidden="true"
                                                         ></span>
-                                                        <span class="sr">${_("This content is graded")}</span>
+                                                        <span class="sr">&nbsp;${_("This content is graded")}</span>
```

Comment:
Just curious what the purpose of the added nbsp is?
```
&nbsp;
```

Reply:
This is to make sure there is a space between the visible text and the screen reader text.  According to the a11y team sometimes if you don't explicitly put a space, the screen reader won't realize there is supposed to be one and try to read things like : "visible contentscreen reader content" instead of "visible content screen reader content"